src/Containers/VendorsList/NavigationBar.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { StyledIcon } from '@styled-icons/styled-icon';
+import { NavigationItem, INavigationItemProps } from './NavigationItem';
+import { MainInterface, ResponsiveInterface } from '../../Utils/BaseStyles';
+import { flex, media, scroll } from '../../Utils/Mixins';
+
+
+export interface INavigationBarItems {
```

Comment:
Why aren't you re-using the type of NavigationBar itself? not sure why creating a new interface here?